pkg/registry/modeljob/util.go

```diff
@@ -19,6 +24,9 @@ func GenerateExtractionModelJob(domain, project, modelName, versionName, format
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      util.RandomNameWithPrefix(fmt.Sprintf("modeljob-%v-%v-%v", project, modelName, versionName)),
 			Namespace: "default",
+			Labels: map[string]string{
+				extractLabelKey: "true",
```

Comment:
Why do we need such a key

Reply:
We have two page for model extraction and model conversion, but it implement by modeljob only, how to list extract task or convert task? so we add a label to filter it in different page.